modules/oe_translation_remote/src/RemoteTranslationRequestEntityTrait.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation_remote;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\oe_translation\LanguageWithStatus;
+use Drupal\oe_translation_remote\Entity\RemoteTranslatorProviderInterface;
+
+/**
+ * Trait for remote translation request entity bundle classes.
+ */
+trait RemoteTranslationRequestEntityTrait {
```

Comment:
Since remote translators get also their own TranslationRequest entity bundle, we have this trait to handle all their communality. In the end we will have the the following bundles: `local`, `epoetry` and `test_remote` (used for testing). The latter 2 will use the trait.